app/src/androidTest/java/com/example/triptracker/view/NavigationBarTest.kt

```diff
@@ -48,6 +48,7 @@ class NavigationBarTest {
         TopLevelDestination(Route.PROFILE, Icons.Outlined.Person, "Profile")
 
     every { navigation.navigateTo(any()) } returns Unit
+    every { navigation.goBack() } returns Unit
```

Comment:
It seems like you are not using this mock in your tests, should we add new tests or remove this line ?

Reply:
This is needed since now when we are in the profile and change to another screen the navigation.goBack() is called first. This implies that the go back is needed but is not shown here

Reply:
Perfect then !